src/OpenTelemetry/Metrics/MetricAggregators/GaugeMetricAggregator.cs

```diff
@@ -19,19 +19,15 @@
 
 namespace OpenTelemetry.Metrics
 {
-    internal class GaugeMetricAggregator : IGaugeMetric, IAggregator
+    public class GaugeMetricAggregator : IGaugeMetric, IAggregator
```

Comment:
is this intentional?

Reply:
Yes. We need users to be able to configure an Aggregator.

Also, if we like, we can create a thin layer for Aggregator "interface"/configuration vs "implementation". It'll be artificial, but may make people more comfortable.

Reply:
Just realized the View Spec PR doesn't specify aggregator in the criteria. That'd be a qn to the spec pr :)

I'd be very hesitant of exposing these classes initially.

Reply:
Resolved with simplified AddView apis.